pyscf/cc/gccsd.py

```diff
@@ -128,7 +128,7 @@ def init_amps(self, eris=None):
         eijab = lib.direct_sum('ia,jb->ijab', eia, eia)
         t1 = eris.fock[:nocc,nocc:] / eia
         eris_oovv = np.array(eris.oovv)
-        t2 = eris_oovv / eijab
+        t2 = eris_oovv.conj() / eijab
         self.emp2 = 0.25*einsum('ijab,ijab', t2, eris_oovv.conj()).real
```

Comment:
Should this line be changed, too? Since t2 is now conjugated, should eri not be conjugated? That is my impression but I've not spent much time reviewing this.

Reply:
Yes. That's an accurate point. Need to remove .conj() from eris_oovv. Let me update the code.
For a confirmation, here is a toy example:
code)
e_mp2 = mymp2.kernel()[0]
e_wc = 0.25*einsum('ijab,ijab', t2, eris_oovv.conj())
e_woc = 0.25*einsum('ijab,ijab', t2, eris_oovv)
print(f'MP2: {e_mp2:.5e}')
print(f'With conjugate: {e_wc:.5e}, Without conjugate: {e_woc:.5e}')
output)
MP2: -1.78100e-03
With conjugate: 1.15364e-04-5.78633e-05j, Without conjugate: -1.78100e-03-4.59033e-23j